File: 01-Ejercicios-Clases-2023_2/C2-08-18-23/Principal.java
class Principal{
	public static void main(String[] args){
		//Persona p1 = new Persona();
		Estudiante e1 = new Estudiante();
		Docente d1 = new Docente();
		//e1.nombre = "Juan";
		e1.setNombre("Juan");
		e1.CRAEST = 99.9f;
		Persona.CRAEST = 100.0f;
		//System.out.println(e1.getNombre());
		e1.imprimirDatos();
		d1.imprimirDatos();
		Persona.mostrarMensaje();
		
		e1.mostrarCodigoPUCP();
		d1.mostrarCodigoPUCP();
	}
}

File: 01-Ejercicios-Clases-2023_2/C4-08-25-23/Ejercicio02/Principal.cs
class Principal{
	public static void Main(){
		Persona p1 = new Persona("Pedro");
		Estudiante e1 = new Estudiante("Manuel",99.9);
		//p1.Nombre = "Pedro";
		System.Console.WriteLine("Mi primer programa C#");
		System.Console.WriteLine(p1.Nombre);
		System.Console.WriteLine(e1.Nombre);
		e1.imprimirDatos();
	}
}